src/pages/admin/reports/components/funds-expenditures-section/components/common/add-income-modal/AddIncomeModal.module.scss

```
@@ -81,6 +81,21 @@
     @include type.body-2-regular;
 }
 
+.disabled-select-placeholder {
+    @include type.body-2-regular;
+    width: 100%;
```

Stylelint (17.6.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)